pinot-common/src/main/java/org/apache/pinot/common/utils/helix/HelixHelper.java

```diff
@@ -552,6 +565,17 @@ public static List<InstanceConfig> getInstancesConfigsWithTag(List<InstanceConfi
     return instancesWithTag;
   }
 
+  public static List<InstanceConfig> getInstancesConfigsWithoutTag(List<InstanceConfig> instanceConfigs, String tag) {
```

Comment:
Suggest changing `tag` to `defaultTag` and add some javadoc explaining this is the tag representing that there is no tag associated with the instance. Same for other methods